config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```
@@ -22,6 +22,33 @@ periodics:
     testgrid-tab-name: capi-test-main
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
+- name: periodic-cluster-api-test-mink8s-main
+  interval: 6h
```

```suggestion
  interval: 1h
```

My bad, I looked at the wrong job when writing which interval we should use